x-pack/test/api_integration/apis/endpoint/resolver.ts

```diff
@@ -222,10 +219,28 @@ export default function resolverAPIIntegrationTests({ getService }: FtrProviderC
           .get(`/api/endpoint/resolver/${entityID}/children?legacyEndpointID=foo`)
           .set(commonHeaders)
           .expect(200);
-        expect(body.pagination.total).to.eql(0);
-        expect(body.pagination.next).to.eql(null);
+        expect(body.pagination.nextChild).to.eql(null);
         expect(body.children).to.be.empty();
       });
     });
+
+    describe('tree endpoint', () => {
```

Comment:
Yeah, so the actual underlying functionality of the calls in this endpoint are really covered by the other tests... so I'm not too concerned with the one happy path test since this endpoint is just a mash up of everything else. That said, I agree in practice, and think we should add more after generating a more full testing dataset. With this in mind, is this a blocker?